tests/basis.ts

```diff
@@ -0,0 +1,25 @@
+import "mocha";
+
+import { expect } from "chai";
+
+import { MarketAggregator } from "../src";
+import { R_BASIS_MINT } from "../src/sources";
+
+describe("Basis Market Staking", () => {
+  it("calculates price of rBASIS vs BASIS", async () => {
```

Comment:
This test should act more like xSTEP etc and not use MarketAggregator. It should test the BasisSource directly. MarketAggregator is covered in its own test.

Reply:
I've re-implemented tests now in this way. Testing vs. expected return value rather than being so integration focused.